core/jvm/src/test/scala/zio/sql/ExprSpec.scala

```diff
@@ -0,0 +1,17 @@
+package zio.sql
+
+import zio.test.Assertion.anything
+import zio.test.{ZIOSpecDefault, assert}
+
+object ExprSpec extends ZIOSpecDefault {
+  import ProductSchema._
+
+  def spec = suite("Aggregate Expression")(
+    test("+ works on integer, Double and BigDecimal columns") {
+      val query = agSelect
+      assert(query)(anything)
```

Comment:
not sure what you're testing here....what's `agSelect` ?

Reply:
Added new enhanced Test AgregationSpec for Postgesql

Reply:
The CI-build seem to be flaky my local build was successful
```37 tests passed. 0 tests failed. 1 tests ignored.
Executed in 6 m 15 s```